api/src/test/java/org/openmrs/module/insuranceclaims/dao/InsuranceClaimsDaoTest.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.insuranceclaims.api.dao;
+
+import org.junit.Ignore;
+import org.junit.Test;
+import org.openmrs.api.UserService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.insuranceclaims.Item;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static org.hamcrest.Matchers.hasProperty;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+/**
+ * It is an integration test (extends BaseModuleContextSensitiveTest), which verifies DAO methods
+ * against the in-memory H2 database. The database is initially loaded with data from
+ * standardTestDataset.xml in openmrs-api. All test methods are executed in transactions, which are
+ * rolled back by the end of each test method.
+ */
+public class InsuranceClaimsDaoTest extends BaseModuleContextSensitiveTest {
+	
+	@Autowired
+	InsuranceClaimsDao dao;
```

Comment:
add private

Reply:
I will rewrite that test in separate PR where I will add the POJO classes with DAOs. Thanks.